extensions/code-variable/README.md

```diff
@@ -0,0 +1,35 @@
+# Code Variable
+
+Generate available code variables.
+
+## Result
+
+Currently supported commands: `xt`, `dt`, `xh`, `zh`,  `cl`.
+
+### 小驼峰命名法：`xt`
+
+![小驼峰命名法](./metadata/code-variable-1.png)
+
+### 大驼峰命名法：`dt`
+
+![大驼峰命名法](./metadata/code-variable-2.png)
+
+### 下划线命名法：`xh`
+
+![下划线命名法](./metadata/code-variable-3.png)
+
+### 中划线命名法：`zh`
+
+![中划线命名法](./metadata/code-variable-4.png)
+
+### 常量命名法：`cl`
+
+![常量命名法](./metadata/code-variable-5.png)
+
+## Special Thanks
+
+This project is based on and inspired by Alfred extension [CodeVar](https://github.com/xudaolong/CodeVar).
+
+## License
+
+[MIT](./LICENSE) License © 2023 [Hongbusi](https://github.com/Hongbusi) 
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.